src/main/java/com/adobe/target/edge/client/service/TelemetryService.java

```diff
@@ -34,6 +36,7 @@
   private final ConcurrentLinkedQueue<TelemetryEntry> storedTelemetries =
       new ConcurrentLinkedQueue<>();
   private static final int STATUS_OK = 200;
```

Comment:
I know this isn't related to your PR, but would be great to replace it by directly using `HttpStatus.SC_OK`